test/jdk/java/text/Format/DecimalFormat/CloneTest.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8354522
+ * @summary Check for cloning interference
+ * @run junit/othervm --add-opens=java.base/java.text=ALL-UNNAMED CloneTest
+ */
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+import java.math.BigDecimal;
+import java.text.DecimalFormat;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CloneTest {
+    private static Field DIGIT_LIST_FIELD;
+    private static Class<?> DIGIT_LIST_CLASS;
+
+    @BeforeAll
+    static void setup() throws Exception {
+        DIGIT_LIST_FIELD = DecimalFormat.class.getDeclaredField("digitList");
+        DIGIT_LIST_FIELD.setAccessible(true);
+
+        DecimalFormat df = new DecimalFormat();
+        Object digitList = DIGIT_LIST_FIELD.get(df);
+
+        DIGIT_LIST_CLASS = digitList.getClass();
+    }
```

Comment:
I'd add some comment saying that this is a whitebox testing, so it might fail if the internal implementation changes (quite unlikely though). And if accessing digitList/data fails, it should throw `jtreg.SkippedException` instead of a real exception in the actual @test methods.